python/src/linalg.cpp

```diff
@@ -210,4 +210,28 @@ void init_linalg(py::module_& parent_module) {
             array([[-2.23607, -3.57771],
                    [0, 0.447214]], dtype=float32)
       )pbdoc");
+  m.def(
+      "svd",
+      &svd,
+      "a"_a,
+      py::kw_only(),
+      "stream"_a = none,
+      R"pbdoc(
+        svd(a: array, *, stream: Union[None, Stream, Device] = None) -> (array, array, array)
+
+        The Singular Value Decomposition (SVD) of the input matrix.
+
+        This function supports arrays with at least 2 dimensions. When the input
+        has more than two dimensions, the function operates in "stacked mode" like
+        numpy.
```

Comment:
```suggestion
        NumPy.
```

Reply:
Could you elaborate here on what is meant by "stacked mode"? I think it would be clearer if it said it treats the last two dimensions as the matrix dimensions and computes the SVD for each matrix independently.

Reply:
I used the same term as the NumPy docs:

> When a is higher-dimensional, SVD is applied in stacked mode as explained below.

I rephrased it anyway.